includes/payment-methods/class-wc-stripe-express-checkout-element.php

```diff
@@ -244,6 +244,33 @@ public function javascript_params() {
 		];
 	}
 
+	/**
+	 * Should ECE use product pricing (vs. cart pricing) in the current context.
+	 *
+	 * For One Page Checkout (OPC), when checkout buttons are enabled, always use cart
+	 * context so discounts/coupons are reflected.
+	 *
+	 * @return bool True to use product pricing; false to use cart totals.
+	 */
+	private function get_is_product_page_for_ece() {
```

Comment:
Can we simplify this name to `is_product_page_for_ece()`? It feels like the leading `get_` is unnecessary and a bit weird to me.
```suggestion
	private function is_product_page_for_ece() {
```

Reply:
Makes sense, updated in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/4757/commits/44d54743bd9cf6fd2a0364f6f03c27045fcd0f77